hw/ip/dcd/data/dcd.hjson

```diff
@@ -0,0 +1,402 @@
+      local:   "true",
+    }
+  ],
+  regwidth: "32",
+  registers: [
+    { name: "adc_en_ctl",
+      desc: "ADC enable control register",
+      hwqe:     "true",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      resval: "0",
+      fields: [
+        { bits: "0",
+          name: "adc_enable",
+          desc: "1’b0: to power down ADC and DCD FSM will enter the reset state; 1’b1: to power up ADC and DCD FSM will start",
+        }
+        { bits: "1",
+          name: "oneshot_mode",
+          desc: "Oneshot mode does not care about the filter value. 1’b0: disable; 1’b1: enable",
+        }
+      ]
+    }
+    { name: "adc_pd_ctl",
+      desc: "ADC PowerDown(PD) control register",
+      hwqe:     "true",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      fields: [
+        { bits: "0",
+          name: "lp_mode",
+          desc: "1’b0: adc_pd is disabled, use adc_sample_ctl. 1'b1: adc_pd is enabled, use both adc_lp_sample_ctl & adc_sample_ctl",
+          resval: "0",
+        }
+        { bits: "7:4",
+          name: "pwrup_time",
+          desc: "ADC power up time. unit in cycles, 200KHz(5us)",
+          resval: "6",
+        }
+        { bits: "31:8",
+          name: "wakeup_time",
+          desc: "How often FSM wakes up from ADC PD mode to take a sample. unit in cycles, 200KHz(5us)",
+          resval: "1600",
+        }
+      ]
+    }
+    { name: "adc_lp_sample_ctl",
+      desc: "ADC Low-Power(LP) sample control register",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe:  "true",
+      fields: [
+        { bits: "7:0",
+          name: "lp_sample_cnt",
+          desc: "The number of samples in low-power mode when the low-power mode is enabled. After the programmed number is met, ADC won’t be powered down any more.",
+          resval: "4",
+        }
+      ],
+    }
+    { name: "adc_sample_ctl",
+      desc: "ADC sample control register",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
+      resval: "155",
+      fields: [
+        { bits: "15:0",
+          name: "np_sample_cnt",
+          desc: "The number of samples in normal-power mode to meet the debounce spec. used after the low-power mode condition is met or in the normal power mode",
+        }
+      ]
+    }
+    { name: "adc_fsm_rst",
+      desc: "ADC FSM reset control",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
+      resval: "0",
+      fields: [
+        { bits: "0",
+          name: "rst_en",
+          desc: "1’b0: Normal functional mode. 1’b1: SW to reset all the FSMs and timers",
+        }
+      ]
+    }
+    { multireg: {
+        name: "adc_chn0_filter_ctl",
+        desc: '''ADC channel0 filter range
+
+	      There are 8 filters to define the potential range(min, max)
+              [11:0]: min value [1:0] RO 0
+              [12]: condition(in the box or out of the box)
+	      [27:16]: max value [17:16] RO 0
+	      each step is 2.148mV
```

Comment:
similar comment on description formatting as on https://github.com/lowRISC/opentitan/pull/4900. it might be good to preview the generated documentation to see whether this gets rendered correctly.

Reply:
to see the preview documentation, please run `./util/build_docs.py --preview`, it will then tell you where to view the output.
Still I would recommend at least aligning the next here, they appear to be off.